capitals.rb

```diff
@@ -150,3 +150,39 @@
     name: "Wyoming",
     capital: "Cheyenne"
 }]
+
+
+  states = states.shuffle
+
```

Comment:
👍 you can also write `states.shuffle!` to the same effect. using `!` with most enums returns a modified array